core/templates/dev/head/editor/state_editor_responses.html

```diff
@@ -77,7 +77,11 @@
                                        on-save-answer-group-feedback="saveDefaultOutcomeFeedback"
                                        on-save-answer-group-dest="saveDefaultOutcomeDest"
                                        is-editable="editabilityService.isEditable()"
+<<<<<<< HEAD
```

Comment:
I have been checking manually. I got an error about another merge issue and I fixed it but this one didn't show up.

Done